templates/default/custom.properties.erb

```diff
@@ -39,6 +43,17 @@ org.opencastproject.admin.email=dev-notify@lists.dce.harvard.edu
 org.opencastproject.security.digest.user=<%= @rest_auth[:user] %>
 org.opencastproject.security.digest.pass=<%= @rest_auth[:pass] %>
 
+# The number of times to retry a request if the nonce expires.
+#org.opencastproject.security.digest.nonce.retries=12
+
+# The configuration property specifying the minimum amount of time in seconds wait before retrying a request after
+# a nonce timeout. Default is 300 seconds (5 minutes).
```

Comment:
What is the nature of these requests? Just asking because 5m seems like a long time to wait before retrying.

Reply:
@lbjay  I don't know the nuances of this timeout. I suspect its the suggested delay on an unexpected digest auth communication issue. Here is the code where that param is used. https://github.com/opencast/opencast/blob/develop/modules/kernel/src/main/java/org/opencastproject/kernel/security/TrustedHttpClientImpl.java#L403